.circleci/config.yml

```diff
@@ -21,7 +21,17 @@ jobs:
           command: |
             git config --global user.email "devhub-deploy@users.noreply.github.com"
             git config --global user.name "Devhub Deployer"
-            cd website && yarn install && USE_SSH=true GIT_USER=git yarn run publish-gh-pages
+            # install Docusaurus and generate file of English strings
+            - cd website && yarn install && yarn run write-translations && cd ..
+            # crowdin install
+            - sudo apt-get install default-jre
```

Comment:
I was worried this install would be too much for every push, but it does match the [recommended approach](https://docusaurus.io/docs/en/translation#automated-file-sync-using-circleci).